src/components/Alert/Alert.js

```diff
@@ -1,16 +1,22 @@
-import { alerts } from '@sparkdesignsystem/spark-core'
 import classnames from 'classnames'
 import PropTypes from 'prop-types'
 import React from 'react'
 import ALERT_VARIANTS from './variants'
-import { sparkComponentClassName } from '../../util'
+import { sparkComponentClassName, sparkClassName } from '../../util'
 import Icon from '../Icon/index'
 
 class Alert extends React.Component {
+  constructor(props) {
```

Comment:
what you want here is just 
`state = {}` no need for a constructor, just don't use 'static'